04 Implementation/src/entities/KKOImpl.java

```diff
@@ -0,0 +1,106 @@
+package entities;
+
+import entities.exceptions.ForaeldereksistererikkeException;
+import entities.exceptions.NavnEksistererException;
+import entities.exceptions.NegativBeloebException;
+
+import java.util.ArrayList;
+
+public class KKOImpl implements KKO, Observable{
+    private ObserverManager observerManager;
+
+    private String navn;
+    double beloeb;
+    private String foraeldersNavn;
+    ArrayList<KKO> efterfoelgerer;
```

Comment:
Bliver aldrig initialiseret.
```suggestion
    ArrayList<KKO> efterfoelgerer = new ArrayList<KKO>();
```
NB: Refactor rename til "efterfoelgere".